libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html

```diff
@@ -57,75 +60,61 @@
         />
       }
 
-      <!-- Execution type -->
-      <div class="chips">
-        <label>{{ t("create.executionType") }}</label>
-        <watt-filter-chip choice name="executionType" value="ACTUAL" [selected]="true">
-          {{ t("executionTypes.ACTUAL") }}
-        </watt-filter-chip>
-        <watt-filter-chip choice name="executionType" value="SIMULATION" [disabled]="true">
-          {{ t("executionTypes.SIMULATION") }}
-        </watt-filter-chip>
-      </div>
-
-      <!-- scheduling -->
-      <div class="chips">
-        <label>{{ t("create.scheduling") }}</label>
-        <watt-filter-chip choice name="schedule" value="NOW" [selected]="true">
-          {{ t("scheduling.NOW") }}
-        </watt-filter-chip>
-        <watt-filter-chip choice name="schedule" value="SCHEDULED" [disabled]="true">
-          {{ t("scheduling.SCHEDULED") }}
-        </watt-filter-chip>
-      </div>
-
-      <!-- Process type -->
-      <watt-dropdown
-        [label]="t('create.calculationType.label')"
-        [formControl]="formGroup.controls.calculationType"
-        [options]="calculationTypesOptions"
-        [showResetOption]="false"
-        [multiple]="false"
-        dhDropdownTranslator
-        translateKey="wholesale.calculations.calculationTypes"
-      />
+      <vater-flex direction="row" gap="l">
+        <!-- Process type -->
+        <watt-dropdown
+          vater
+          fill="horizontal"
+          [label]="t('create.calculationType.label')"
+          [formControl]="formGroup.controls.calculationType"
+          [options]="calculationTypesOptions"
+          [showResetOption]="false"
+          [multiple]="false"
+          dhDropdownTranslator
+          translateKey="wholesale.calculations.calculationTypes"
+        />
 
-      <!-- Monthly summation -->
-      <div class="chips">
-        <label>{{ t("create.summation") }}</label>
-        <watt-filter-chip choice name="summation" value="yes" [disabled]="true">
-          {{ t("summation.YES") }}
-        </watt-filter-chip>
-        <watt-filter-chip choice name="summation" value="no" [disabled]="true">
-          {{ t("summation.NO") }}
-        </watt-filter-chip>
-      </div>
+        <!-- Execution type -->
+        <vater-stack gap="s" fill="horizontal" align="flex-start">
+          <label>{{ t("create.executionType") }}</label>
+          <vater-stack direction="row" gap="s" align="center" style="height: 44px">
+            <watt-filter-chip choice name="executionType" value="ACTUAL" [selected]="true">
+              {{ t("executionTypes.ACTUAL") }}
+            </watt-filter-chip>
+            <watt-filter-chip choice name="executionType" value="SIMULATION" [disabled]="true">
```

Comment:
Question: Should value be translated?

Reply:
No, but it probably should come from an enum - problem is that we do not yet have this functionality, so this is just a visual placeholder